ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -205,15 +205,19 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-hypershift
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
-- as: aws-ipi-private-shared-vpc-phz-sts-f28
+- as: aws-ipi-private-shared-vpc-phz-sts-canary-f28
   cron: 32 19 16 * *
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       EXTRACT_MANIFEST_INCLUDED: "true"
+      FILTERS_ADDITIONAL: ~TELCO&
```

Comment:
Instead of adding the TELCO filter for every canary job (which needs to add it multiple times, and we can easily miss one of them), we can add the filter in the canary test chain ( which only need to add it once).